traffic_ops/traffic_ops_golang/crconfig/deliveryservice.go

```diff
@@ -43,14 +43,33 @@ const DefaultTLDTTLNS = 3600 * time.Second
 const GeoProviderMaxmindStr = "maxmindGeolocationService"
 const GeoProviderNeustarStr = "neustarGeolocationService"
 
-func makeDSes(cdn string, domain string, tx *sql.Tx) (map[string]tc.CRConfigDeliveryService, error) {
+func makeDSes(cdn string, domain string, ttlOverride int, tx *sql.Tx) (map[string]tc.CRConfigDeliveryService, error) {
 	dses := map[string]tc.CRConfigDeliveryService{}
 
 	admin := CDNSOAAdmin
-	expireSecondsStr := strconv.Itoa(int(CDNSOAExpire / time.Second))
-	minimumSecondsStr := strconv.Itoa(int(CDNSOAMinimum / time.Second))
-	refreshSecondsStr := strconv.Itoa(int(CDNSOARefresh / time.Second))
-	retrySecondsStr := strconv.Itoa(int(CDNSOARetry / time.Second))
+	expireSeconds := int(CDNSOAExpire / time.Second)
+	minimumSeconds := int(CDNSOAMinimum / time.Second)
+	refreshSeconds := int(CDNSOARefresh / time.Second)
+	retrySeconds := int(CDNSOARetry / time.Second)
+	if ttlOverride > 0 {
+		if ttlOverride < expireSeconds {
```

Comment:
You don't need to change it if you don't want to, but maybe using the `math.min` function reduces the lines of code here? I'm fine either way.

Reply:
Using `math.Min()` in f1340e3d56